tests/openadapt/test_scrub.py

```diff
@@ -0,0 +1,223 @@
+"""Module to test scrub.py"""
+
+from io import BytesIO
+import os
+import warnings
+
+from PIL import Image
+
+from openadapt import scrub, config
+
+
+def test_scrub_image() -> None:
```

Comment:
Please add a `config.SCRUB_ENABLED` parameter that disables scrubbing everywhere

Reply:
Done!